source/developers-guide/event-list/index.md

```diff
@@ -1354,3 +1354,65 @@ stdClass Object
         )
 )
 ```
+
+## Theme_Inheritance_Template_Directories_Collected
+
+### Definition
+```
+public function getTemplateDirectories(Shop\Template $template)
+{
+    $directories = $this->getTemplateDirectoriesRecursive(
+        $template->getId(),
+        $this->fetchTemplates()
+    );
+
+    $directories = $this->eventManager->filter(
+        'Theme_Inheritance_Template_Directories_Collected',
+        $directories,
+        ['template' => $template]
+    );
+
+    return $directories;
+}
+```
+
+### Registration
+```
+public static function getSubscribedEvents()
+{
+    return [
+        'Theme_Inheritance_Template_Directories_Collected' => 'onCollectTemplateDirectories',
+    ];
+}
+```
+
+### Listener
+```
+public function onCollectTemplateDirectories(\Enlight_Event_EventArgs $args)
+{
+    /** @var $directories array */
+    $directories = $args->getReturn();
+
+    /** @var $template \Shopware\Models\Shop\Template */
+    $template = $args->get('template');
+
```

Comment:
a short example, how to set a new directory would be nice. because it is not similar to the known `addTemplateDir` thing